### PR TITLE
Made Footer Revealable

### PR DESCRIPTION
**Pull request by MridulTi**
Solved the Issue #347
I made the Footer Revealable for all pages except Talking Therapy. I also added a div in Loading Id as it goes affected due to position: sticky I used in the Footer class.
Earlier Footer:-
image

Updated revealable Footer:-
https://drive.google.com/file/d/1TuEVDtHxvnJZ5sLsnwn3vTVxHsI4U_hN/view?usp=sharing

### CODE REVIEW

**Review by myhacktech — APPROVED** [review]

File: html/yogatherapy.html [hunk 1]
```diff
@@ -75,34 +76,51 @@ <h1>Yoga Enhances Your Life</h1>
...
-                    <div class="yoga-text"><i class="fa fa-dot-circle-o" aria-hidden="true"></i><h3>Yoga Increases <strong>Energy</strong></h3></div>
-                    <div class="yoga-text"><i class="fa fa-dot-circle-o" aria-hidden="true"></i><h3>Yoga Gives you Better <strong>Flexibility and Posture</strong></h3></div>
-                    <div class="yoga-text"><i class="fa fa-dot-circle-o" aria-hidden="true"></i><h3>Yoga helps in improving <strong>intuition</strong></h3></div>
-            </div>
-
+      <div class="col-md-12 text-center">
+        <h1><strong>Benefits of Yoga</strong></h1>
+      </div>
+      <div class="bfit">
+        <img src="https://preview.colorlib.com/theme/yogabest/images/xabout.png.pagespeed.ic.H7uxRdWOgu.webp" width="700"
```
Comment (AliReza1083) [inline html/yogatherapy.html:86]: The code seems to be amazing but it's better to write the **width** and **height** of an image in CSS file.

``` diff
- <img src="" width="700" height="600">
+ img { width: 700px; height: 600px;}
```

> **Note**
> Why should we use it in CSS file?
1. For making the code to looks clear for code reader.
2. For making Responsive

File: html/yogatherapy.html [hunk 2]
```diff
@@ -133,10 +149,8 @@ <h1>Virabhadrasana</h1>
               case of frozen shoulders. Know the steps of doing the posture, all
               its benefits, and contraindications by cicking below.
             </p>
-            <a
-              href="https://www.artofliving.org/in-en/yoga/yoga-poses/warrior-pose-virbhadrasana"
-              ><button>Read More</button></a
-            >
+            <a href="https://www.artofliving.org/in-en/yoga/yoga-poses/warrior-pose-virbhadrasana"><button>Read
```
Comment (AliReza1083) [inline html/yogatherapy.html:152]: If you are using a tag, There is no need use **_button_** inside of it.
You can design it to look like a button.

``` diff
- <a href=""><button></button></a>
+ <a href=""></a>
```

> **Note**
> For styling a tag, If you want to add `padding` on it, set the display to equal to `inline-block`.

>**Warning**
> If you don't use `inline-block`, The `padding-top` and `padding-bottom` is not going to work.

**Review by AliReza1083 — COMMENTED** [review]
If You follow this step, Your code will look nicer.
Wish you all the best.

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]
@MridulTi Your PR would break the whole website. Kindly check the deploy link. It's horrible. Also, your PR has merge conflicts.

I don't think we should approve this PR and merge it.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Brittle, Not maintainable

**Comment by MridulTi** [comment]
> t maintainable

I don't understand as to why there is merge conflict even after I resolved it once

**Pull request closed without merging**